pom.xml

```diff
@@ -401,7 +408,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>3.2.4</version>
+                <version>3.3.1-SNAPSHOT</version>
```

Comment:
We should not be using snapshot versions without there being a good reason, and that reason should be being documented here as a comment. (In the POM, Not Github comments)